src/main/java/com/atikinbtw/velocitycoollist/VelocityCoolList.java

```diff
@@ -71,7 +71,9 @@ public void onProxyInitialization(ProxyInitializeEvent event) {
         commandManager.register(commandMeta, commandToRegister);
 
         logger.info("VelocityCoolList has been enabled!" + " Took " + (System.currentTimeMillis() - startTime.getTime()) + "ms");
-        checkForUpdates();
+        if ((boolean) Config.get("autoupdate", true)) {
+            checkForUpdates();
+        }
```

Comment:
again, same, i would rename this to update checker

Reply:
fixed also in [f6f168f](https://github.com/atikiNBTW/VelocityCoolList/pull/3/commits/f6f168f7c9ee74a15073924f43fb8fbcaee278bd)